src/tracing/esp32_trace/BUILD.gn

```diff
@@ -20,8 +20,12 @@ config("tracing") {
   include_dirs = [ "include" ]
 }
 
-source_set("esp32_trace") {
-  public = [ "include/matter/tracing/macros_impl.h" ]
-  sources = [ "include/matter/tracing/macros_impl.cpp" ]
+source_set("esp32_trace_tracing") {
+  public = [ "include/matter/tracing/macros_impl.h",
```

Comment:
macros_impl should be in a include folder (this is what the other places do and this is what the config above says it should be)

Reply:
Mis-read here ... the header is in the right place, but the rest should not be here.